docs/source/tutorial.rst

```diff
@@ -44,20 +43,28 @@ We then proceed by initializing the data space within a Python script called ``i
     # init.py
     import signac
 
-    project = signac.init_project("ideal-gas-project")
+    project = signac.init_project()
 
     for p in range(1, 10):
         sp = {"p": p, "kT": 1.0, "N": 1000}
-        job = project.open_job(sp)
-        job.init()
+        job = project.open_job(sp).init()
 
-The :py:func:`signac.init_project` function initializes the **signac** project in the current working directory by creating a configuration file called ``signac.rc``.
-The location of this file defines the *project root directory*.
-We can access the project interface from anywhere within and below the root directory by calling the :py:func:`signac.get_project` function, or from outside this directory by providing an explicit path, *e.g.*, ``signac.get_project('~/ideal_gas_project')``.
+The :py:func:`signac.init_project` function initializes the **signac** project in the current working directory by creating a hidden ``.signac`` subdirectory.
+The location of this directory defines the *project root directory*.
+Initially, the ``.signac`` directory will contain the minimal configuration information required to define the project.
 
-.. note::
+.. code:: bash
 
-    The name of the project stored in the configuration file is independent of the directory name it resides in.
+    ~/ideal_gas_project $ python init.py
+    ~/ideal_gas_project $ ls -a
+    .               ..              .signac         init.py         workspace
+    ~/ideal_gas_project $ ls .signac
+    config
+    ~/ideal_gas_project $ cat .signac/config
+    schema_version = 2
+
+
+We can access the project interface from anywhere within and below the root directory by calling the :py:func:`signac.get_project` function, or from outside this directory by providing an explicit path, *e.g.*, ``signac.get_project('~/ideal_gas_project')``.
```

Comment:
Will the `~` be automatically expanded? i.e. Do we call `os.path.expanduser`?

Reply:
Good catch, no we do not. For now let's just remove the `~/` and use an example e.g. `signac.get_project('/path/to/ideal_gas_project/')`